File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.cc
#include <stdio.h>
int arr[101] = {0,};
int main(void) {
    int n, m;
    scanf("%d %d", &n, &m);

    for (int i = 1; i <= n; i++) arr[i] = i;

    for (int i = 0; i < m; i++) {
        int a, b;
        scanf("%d %d", &a, &b);

        int tmp = arr[a];
        arr[a] = arr[b];
        arr[b] = tmp;    
    }

    for (int i = 1; i <= n; i++) printf("%d ", arr[i]);
    return 0;
}

File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int m = sc.nextInt();

        int arr[] = new int[n+1];
        for (int idx = 1; idx <= n; idx++) arr[idx] = idx;

        int i, j;
        for (int t = 0; t < m; t++) {
            i = sc.nextInt();
            j = sc.nextInt();

            int tmp = arr[i];
            arr[i] = arr[j];
            arr[j] = tmp;
        }

        for (int idx = 1; idx <= n; idx++) System.out.print(arr[idx] + " ");
    }
}